pom.xml

```diff
@@ -59,8 +59,8 @@ agnostic home for software distribution comprehension and audit tools.
     <mavenJavadocPluginVersion>3.5.0</mavenJavadocPluginVersion>
     <mavenPmdPluginVersion>3.20.0</mavenPmdPluginVersion>
     <!-- Used to generate download page for RAT during site builds, please adapt versions manually BEFORE doing a release -->
-    <previousRatVersion>0.16</previousRatVersion>
-    <currentSnapshotRatVersion>0.17-SNAPSHOT</currentSnapshotRatVersion>
+    <previousRatVersion>0.15</previousRatVersion>
+    <currentSnapshotRatVersion>0.16-SNAPSHOT</currentSnapshotRatVersion>
```

Comment:
These versions are a preparation for an upcoming release and should not be changed - they ensure that the site build during a release contains the correct version links.